test/test_compile.py

```diff
@@ -0,0 +1,28 @@
+import torch 
+import time 
+
+class ToyModel(torch.nn.Module):
+    def __init__(self):
+        super(ToyModel, self).__init__()
+        self.linear = torch.nn.Linear(10, 10)
+    def forward(self, x):
+        return self.linear(x)
+
+def test_compiled_model_can_be_saved():
+    model = ToyModel()
+
+    tic = time.time()
+    model(torch.randn(1, 10))
+    toc = time.time()
+    first_inference_duration = toc - tic
+
+    model.compile()
+    tic = time.time()
+    model(torch.randn(1, 10))
+    toc = time.time()
+    compilation_duration = toc - tic 
+
+    assert compilation_duration > first_inference_duration
```

Comment:
Note that we can't keep that in the final test ;)

Reply:
Hmm yeah agree this might be flaky - I guess instead I can just `getattr` and see if the optimized module is on the class

Reply:
Yes. Or capture some dynamo log/behavior to make sure it runs.